docs/build/build-run-package-cwagent-binaries.md

```diff
@@ -0,0 +1,41 @@
+## Build, package, run Amazon CloudWatch Agent's binaries
```

Comment:
Minor nitpick, we can remove one `#` from all of these headers if this is going to be in a separate file.